src/pyspextool/telluric/load_spectra.py

```diff
@@ -277,13 +294,23 @@ def load_data():
         message = 'The standard lacks an order the object has.'
         raise pySpextoolError(message)
 
+    # Determine the angular separation between the object and standard
+
+    obj_long = ten(object_hdrinfo['RA'][0])*15*np.pi/180.
+    obj_lat = ten(object_hdrinfo['DEC'][0])*np.pi/180.
+    std_long = ten(standard_hdrinfo['RA'][0])*15*np.pi/180.
+    std_lat = ten(standard_hdrinfo['DEC'][0])*np.pi/180.   
+
+    angle = angular_separation(obj_long,obj_lat,std_long,std_lat)*180/np.pi
```

Comment:
Use astropy units to convert to degrees. https://docs.astropy.org/en/stable/coordinates/angles.html

Reply:
I have tried to avoid using other packages as much as possible. Things change, and this way if I write it, then I know what it does. Obviously that doesn't work for everything, but for this kind of stuff I want to do it this way.